package.json

```diff
@@ -20,13 +20,13 @@
   "homepage": "https://github.com/matthew-andrews/isomorphic-fetch/issues",
   "dependencies": {
     "node-fetch": "^1.0.1",
-    "whatwg-fetch": ">=0.10.0"
+    "whatwg-fetch": "niftylettuce/fetch"
```

Comment:
don't accept this PR till I look into that jsdom#1203 more, thanks @matthew-andrews